docs/general/administration/troubleshooting.md

```diff
@@ -130,3 +130,51 @@ After opening the database, navigate to the Execute SQL Tab and execute the foll
 UPDATE Users SET InvalidLoginAttemptCount = 0 WHERE Username = 'LockedUserName';
 UPDATE Permissions SET Value = 0 WHERE Kind = 2 AND UserId IN (SELECT Id FROM Users WHERE Username = 'LockedUserName');
 ```
+
+## Fix Admin User Permissions
+
+If the Permissions for your Admin Account break, you can Restore them using simple SQL Queries.
+
+:::caution
+Manual changes to the database can destroy your Instance beyond repair. to prevent this create a copy of your database before executing:
```

Comment:
```suggestion
Manual changes to the database can destroy your instance beyond repair. to prevent this create a copy of your database before executing:
```